src/Okta.Sdk/LogsClient.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Okta.Sdk
+{
+    public sealed partial class LogsClient : OktaClient, ILogsClient, IAsyncEnumerable<ILogEvent>
+    {
+        public IAsyncEnumerator<ILogEvent> GetEnumerator()
```

Comment:
This should call GetLogs, similar to: https://github.com/okta/okta-sdk-dotnet/blob/master/src/Okta.Sdk/UsersClient.cs#L19

Reply:
This allows the developer to do this:

```csharp
var logs = await client.Logs.ToArray();
```

instead of

```csharp
var logs = await client.Logs.GetLogs().ToArray();
```